lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -3,7 +3,7 @@
  * Licensed under the MIT License.
  */
 
-import { AccountInfo, AuthenticationResult, SilentFlowRequest } from "@azure/msal-common";
+import { AccountInfo, AuthenticationResult, AuthorizationUrlRequest, PromptValue, SilentFlowRequest } from "@azure/msal-common";
```

Comment:
Once you update from dev these request types should be imported from within browser, FYI

Reply:
done.